packages/service/tests/unit/test_decoder.py

```diff
@@ -176,7 +176,7 @@ def _get_grpc_serialized_data(values):
 def _get_test_parameter_by_id(default_values):
     parameter_by_id = {
         1: ParameterMetadata.initialize(
-            display_name="float_data!",
+            display_name="float_data",
```

Comment:
If this is valid, why are you changing it?

Reply:
Originally it didn't have a exclamation mark but for testing purposes I put it there., then I forgot to remove it after finishing whatever I was doing.

Reply:
ok, I see